assets/src/apps/authoring/components/EditingCanvas/EditingCanvas.tsx

```diff
@@ -114,6 +114,7 @@ const EditingCanvas: React.FC = () => {
   // TODO: rename first param to partId
   const handlePartConfigure = async (part: any, context: any) => {
     /* console.log('[handlePartConfigure]', { part, context }); */
+    dispatch(setCurrentPartPropertyFocus({ focus: false }));
```

Comment:
These changes are related to https://github.com/Simon-Initiative/oli-torus/pull/5211